vmdb/app/models/ems_kubernetes.rb

```diff
@@ -3,6 +3,7 @@ class EmsKubernetes < EmsContainer
   has_many :container_groups,                     :foreign_key => :ems_id, :dependent => :destroy
   has_many :container_services,                   :foreign_key => :ems_id, :dependent => :destroy
   has_many :container_replication_controllers,    :foreign_key => :ems_id, :dependent => :destroy
+  has_many :container_namespaces,                 :foreign_key => :ems_id, :dependent => :destroy
```

Comment:
We'll have to sync with the openshift patch on this one. It's not worth having 2 different tables (`container_projects` and `contaner_namespaces`) that are essentially the same and hold almost entirely the same data.